### ISSUE TITLE
createdb instruction in documentation is unclear or incorrect

### ISSUE THREAD

**Issue by b1sar** [cookiecutter/cookiecutter-django#3605]
## What happened?
While following the tutorial on [Getting Up and Running](https://cookiecutter-django.readthedocs.io/en/latest/developing-locally.html) in the step 5 (Create a new PostgreSQL database using [createdb](https://www.postgresql.org/docs/current/static/app-createdb.html):) , I tried to run the command as described there,  I got the following error:
```powershell
(.venv) D:\path\to\my\project\directory>createdb my_project_slug -U postgres --password mypassword
createdb: error: too many command-line arguments (first is "postgres")
Try "createdb --help" for more information.
```

Tried different argument combinations where the project slug was the first argument to the `createdb `but every time I got the same error message.
## What should've happened instead?
Looking at the [PostgreSQL creadedb documentation](https://www.postgresql.org/docs/current/app-createdb.html), as stated in the synopsis, the dbname argument,which in the cookiecutter context is equivalent of project_slug, should have been the last argument to the createdb. 

So In the cookiecutter-django documentation in the step 5 of the mentioned part, instead of giving a instruction like this:
```bash

$ createdb <what you have entered as the project_slug at setup stage> -U postgres --password<password>
```

It could be like as following (the project_slug is the last argument here):
```bash
$ createdb -U postgres --password <password> <what you have entered as the project_slug at setup stage>
```

Or in my opinion, more concise way to put it could be as following:

```bash
$ createdb -U postgres --password=<password> <project_slug>

Note: project_slug is what you have entered as the project_slug at setup stage
```
first giving a keyword, then its explanation.

## Additional details

* Host system configuration:
    * Version of cookiecutter CLI (get it with `cookiecutter --version`):
    * OS name and version:

      ```bash
      OS Name:                   Microsoft Windows 10 Home Single Language
      OS Version:                10.0.19043 N/A Build 19043
      ```

    * Python version, run `python3 -V`: `Python 3.9.2`

* Options selected and/or [replay file](https://cookiecutter.readthedocs.io/en/latest/advanced/replay.html):
  On Linux and MacOS: `cat ${HOME}/.cookiecutter_replay/cookiecutter-django.json`
  (Please, take care to remove sensitive information)
    ```json
    {
  "cookiecutter": {
    "version": "0.1.0",
    "timezone": "UTC",
    "windows": "y",
    "use_pycharm": "n",
    "use_docker": "y",
    "postgresql_version": "14.1",
    "js_task_runner": "None",
    "cloud_provider": "AWS",
    "mail_service": "Mailgun",
    "use_async": "n",
    "use_drf": "n",
    "custom_bootstrap_compilation": "n",
    "use_compressor": "n",
    "use_celery": "n",
    "use_mailhog": "n",
    "use_sentry": "n",
    "use_whitenoise": "n",
    "use_heroku": "n",
    "ci_tool": "None",
    "keep_local_envs_in_vcs": "y",
    "debug": "y",
    "_template": "https://github.com/cookiecutter/cookiecutter-django"
  }
  }
    ```
<summary>
Logs:
<details>
<pre>
$ cookiecutter https://github.com/cookiecutter/cookiecutter-django
project_name [Project Name]: ...
</pre>
</details>
</summary>

**Comment by browniebroke** [comment]
Weird, I'm pretty sure it used to work, maybe something changed in the syntax of the command. Anyway, great suggestion, pull request welcome!

**Comment by b1sar** [comment]
To make sure, I checked the history. The last commit that edited that part seems to be this commit 53940411cc96f21f9ed77b3ff241844eb0b2c57d , the `-U postgres --password <password>` part was added in this commit.

But according to the Postgresql documentation there is no need to use `--password` option at all because it will pop up anyways. Also if the `PGUSER=postgres` environment is present there is no need to use `-U postgres` too. But it would be convenient to have that option on condition that `dbname `is the last argument.

Thank you. I'll test and create a pull request.
